internal/robot/ratelimit.go

```diff
@@ -0,0 +1,69 @@
+package robot
+
+import (
+	"strings"
+	"time"
+
+	robotmodels "github.com/syself/hrobot-go/models"
+)
+
+func InitRateLimit(rateLimitWaitTime time.Duration) {
+	rateLimit = rateLimitHandler{
+		waitTime: rateLimitWaitTime,
+	}
+}
```

Comment:
What about using https://pkg.go.dev/k8s.io/client-go@v0.28.3/util/flowcontrol#Backoff ?

Reply:
This is using an exponential backoff instead of the constant backoff we have right now. I am tempted to just remove the whole Robot ratelimiting section for now and push it to a follow up PR (where we might also rething Hetzner Cloud rate limiting, which we do not handle at all right now).

Reply:
Removed it from this Patch series, will submit a follow up PR to add it back in in a less invasive manner using standard tooling.